contrib/debian/rules

```diff
@@ -70,6 +70,11 @@ override_dh_install:
 	rm -f debian/fwupd/usr/lib/*/fwupd-plugins-*/libfu_plugin_test_ble.so
 	rm -f debian/fwupd/usr/lib/*/fwupd-plugins-*/libfu_plugin_invalid.so
 	rm -f debian/fwupd/etc/fwupd/remotes.d/fwupd-tests.conf
+	mv debian/fwupd/usr/share/man/man1 debian/fwupd/usr/share/man/man8
+	for file in debian/fwupd/usr/share/man/man8/*.1; do \
+		file8=$${file%%.1}.8; \
+		mv -v $$file $$file8; \
```

Comment:
the logic behind this is https://lintian.debian.org/tags/manual-page-for-system-command?version=2.104.244